central/integrationhealth/store/store.go

```diff
@@ -0,0 +1,23 @@
+package store
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/generated/storage"
+)
+
+// Store encapsulates the integration health store
+type Store interface {
```

Comment:
nit: The common `Store` interface could be simplified to only expose the functions that are actually used in the datastore layer.
That is `Get`, `Upsert`, `Delete` and `Walk` if I didn't miss anything.

Reply:
Yes, good point, will replace.